packages/services-store/src/actions/profileAPI.ts

```diff
@@ -40,8 +40,8 @@ export type ProfileAPIActions = ReturnType<typeof setMonitorUserStatusError> | R
  * @returns A Redux action that monitors user authentication status.
  */
 export function monitorUserStatus(): ThunkAction<void, { profileAPI: ProfileAPIState }, void, ProfileAPIActions> {
-  return dispatch => {
-    ProfileAPI.monitorUserStatus((error: Error, status: UserStatus) => {
+  return async dispatch => {
+    await ProfileAPI.monitorUserStatus((error: Error, status: UserStatus) => {
```

Comment:
Seems that this PR changes `ProfileAPI.monitorUserStatus()` from a callback-based function to an async function. And thus the call here should be changed accordingly?

Reply:
Oh sorry didn't realize we had `ProfileAPI.getUserStatus()` already... My apologies. In this case, we can rename `monitorUserStatus()` Redux action to `getUserStatus()`, change this implementation to use `ProfileAPI.getUserStatus()` (instead of `ProfileAPI.monitorUserStatus()`) and change `masthead-container.ts` and `masthead-composite.ts` accordingly.